cdap-ui/app/directives/my-pipeline-configurations/my-realtime-pipeline-config/my-realtime-pipeline-config.html

```diff
@@ -260,7 +260,7 @@ <h3 class="modeless-title">
               Client
             </span>
             <i class="fa fa-info-circle"
-                uib-tooltip="Resources for the Apache Spark client process"
+                uib-tooltip="Resources for the client process"
```

Comment:
Resources for client process which launches the Apache Spark Streaming pipeline.